todoman/cli.py

```diff
@@ -309,19 +309,13 @@ def cli(click_ctx, colour, porcelain, humanize, config):
     locale.setlocale(locale.LC_TIME, "")
 
     if not click_ctx.invoked_subcommand:
-        invoke_command(
-            click_ctx,
-            ctx.config['main']['default_command'],
-        )
+        invoke_command(ctx.config['main']['default_command'])
 
 
-def invoke_command(click_ctx, command):
-    name, *args = command.split(' ')
-    if name not in cli.commands:
-        raise click.ClickException(
-            'Invalid setting for [main][default_command]'
-        )
-    click_ctx.invoke(cli.commands[command], args)
+def invoke_command(argv):
+    if not isinstance(argv, type([])):
```

Comment:
`list` wasn't working here since there's a `def list` below. I'm not super familiar with python, but is there a way to reference the builtin `list` type rather than needing to do `type([])`? I'm also not a huge fan of the `type([])`

Reply:
Oooh. Yeah, I blame myself for naming that function `list`. Current versions of `click`allow having a command called `list` but naming the function something different.

You can leave it as it is now, I can address this separately by also renaming that function. :)